dggbot/message.py

```diff
@@ -66,7 +66,10 @@ class PollMessage(Message):
     totals: List[int] = None
```

Comment:
Replace List[int] with list[int] (lowercase, python 3.9 made it so you don't need to import List from the typing module).